docs/migration_guide.rst

```diff
@@ -1,6 +1,32 @@
 Migration guide
 ===============
 
+**********************
+0.21.0 Migration guide
+**********************
+
+Version 0.21.0 of **starknet.py** comes with support for RPC 0.7.0!
+
+0.21.0 Targeted versions
+------------------------
+
+- Starknet - `0.13.1 <https://docs.starknet.io/documentation/starknet_versions/version_notes/#version0.13.1>`_
+- RPC - `0.7.0 <https://github.com/starkware-libs/starknet-specs/releases/tag/v0.7.0>`_
+
+0.21.0 Breaking changes
+-----------------------
+
+.. currentmodule:: starknet_py.net.client_models
+
+1. :class:`ExecutionResources` has a new required field ``data_availability``
+2. :class:`InvokeTransactionTrace`, :class:`DeclareTransactionTrace` and :class:`DeployAccountTransactionTrace` have a new required field ``execution_resources``
```

Comment:
Should we consider addition of new fields changes breaking, assuming that we don't instantiate those manually?

Reply:
Good question. To me it makes sense to classify the addition of new fields as a minor change and to consider the removal of existing keys a breaking change. @THenry14 wdyt?

Reply:
yup, makes sense to me as well